lib/styles/editor/_menu.scss

```diff
@@ -147,18 +147,20 @@
       flex-wrap: wrap;
       gap: 8px;
 
-      .neeto-editor-variable {
-        padding: 4px 6px;
-        font-size: var(--neeto-ui-text-xs);
-        color: rgb(var(--neeto-ui-gray-700));
-        line-height: 1;
-        background-color: rgb(var(--neeto-ui-pastel-blue));
-        border-radius: var(--neeto-ui-rounded-sm);
-        transition: var(--neeto-ui-transition);
-
-        &:hover {
-          background-color: rgb(var(--neeto-ui-info-100));
-        }
+      button {
+        padding: 6px;
+        border: solid 1px rgb(var(--neeto-ui-gray-300));
+        border-radius: var(--neeto-ui-rounded-md);
+        user-select: none;
+      }
+
+      button:hover {
```

Comment:
Combine the selectors for `button:hover`, `button:focus` and `button:focus-visible` and apply the following rules.

```
    button:focus-visible,
    button:focus,
    button:hover{
      color: rgb(var(--neeto-ui-primary-800));
      border-color: rgb(var(--neeto-ui-primary-800));
      outline: none;
      box-shadow: none;
    }
```